### PR TITLE
--admin-enabled no longer requires a input value

### PR DESCRIPTION
**Pull request by shhsu**
Fixing

https://github.com/Azure/azure-cli/issues/1897

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shhsu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shhsu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 1]
```diff
@@ -32,6 +32,10 @@
 register_cli_argument('acr', 'location', location_type)
 register_cli_argument('acr', 'tags', tags_type)
 register_cli_argument('acr', 'admin_enabled',
+                      nargs='?',
+                      required=False,
+                      const='true',
+                      default=None,
                       help='Indicates whether the admin user is enabled',
                       choices=['true', 'false'])
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: By convention, we tend to keep these all on a single line.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def acr_update_get(client,
     )
 
 def acr_update_custom(instance,
-                      admin_enabled=None,
+                      admin_enabled,
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py]: This should still be admin_enabled=None because on updates, "None" means "leave this alone".

What you can do is:
```Python
register_cli_argument('acr create', 'admin_enabled', nargs='?', ..., choices=['true', 'false'])
register_cli_argument('acr update', 'admin_enabled', nargs='1', ..., choices=['true', 'false'])
```

Comment (shhsu) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py]: Do we want to disallow
"acr update --admin-enabled"
but allow
"acr crate --admin-enabled"

Would that be confusing?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py (hunk 1 above)
Comment (shhsu) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: This is a formatting change that would apply to the whole file. If it's needed let's do it in another PR.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py (hunk 2 above)
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py]: Everywhere else in the CLI, the create exposes a flag and the update a binary choice list (either true/false or enabled/disabled).  So I don't think it would be confusing. The primary things on updates is that you leave everything as is except what was specifically specified.

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py [hunk 3]
```diff
@@ -20,30 +20,18 @@
...
 
-register_cli_argument('acr', 'registry_name',
-                      options_list=('--name', '-n'),
-                      help='The name of the container registry',
-                      completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
-register_cli_argument('acr', 'storage_account_name',
-                      help='The name of an existing storage account',
-                      completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
+register_cli_argument('acr', 'registry_name', options_list=('--name', '-n'), help='The name of the container registry', completer=get_resource_name_completion_list(ACR_RESOURCE_TYPE))
+register_cli_argument('acr', 'storage_account_name', help='The name of an existing storage account', completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
```
Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: Can you revert these unnecessary style changes since they are breaking pylint.

Comment (djyou) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: Or add "#pylint: disable=line-too-long" to disable line-too-long checks in this file.

Comment (shhsu) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py]: fixed, thanks

**Comment by shhsu** [comment]
Thanks for approving. I usually see a merge/squash-and-merge button on the page but I don't have one here.

I suspect that I have no write access to the branch. Could someone add me ore merge the PR for me?
